pkg/utils/utils.go

```diff
@@ -22,30 +22,14 @@ func GetBytes(key interface{}) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-// ParseURL extracts database parameters from a string as a URL
-//   bolt://storj.db
-//   bolt://C:\storj.db
-//   redis://hostname
-func ParseURL(s string) (*url.URL, error) {
-	if strings.HasPrefix(s, "bolt://") {
-		return &url.URL{
-			Scheme: "bolt",
-			Path:   strings.TrimPrefix(s, "bolt://"),
-		}, nil
+// SplitURL returns the schema and non-scheme portions of a URL
+// consider https://github.com/xo/dburl if this ends up lacking
+func SplitURL(s string) (string, string, error) {
+	parts := strings.SplitN(s, "://", 2)
+	if len(parts) != 2 {
+		return "", "", fmt.Errorf("Could not parse DB URL %s", s)
 	}
-	if strings.HasPrefix(s, "sqlite3://") {
-		return &url.URL{
-			Scheme: "sqlite3",
-			Path:   strings.TrimPrefix(s, "sqlite3://"),
-		}, nil
-	}
-	if strings.HasPrefix(s, "postgres://") {
-		return &url.URL{
-			Scheme: "postgres",
-			Path:   s,
-		}, nil
-	}
-	return url.Parse(s)
+	return parts[0], parts[1], nil
```

Comment:
I'm not sure if this will work for postgres, Something like this doesn't work for me:
`dbx.Open("postgres", "postgres:abc123@localhost/storj?sslmode=disable")`
but this works correctly:
`dbx.Open("postgres", "postgres://postgres:abc123@localhost/storj?sslmode=disable")`

Reply:
fixed